CONTRIBUTING.md

```diff
@@ -7,12 +7,38 @@ start participating.
 
 ## Topics
 
+* [Development environment](#development-environment)
 * [Reporting Security Issues](#reporting-security-issues)
 * [Design and Cleanup Proposals](#design-and-cleanup-proposals)
 * [Reporting Issues](#reporting-other-issues)
 * [Quick Contribution Tips and Guidelines](#quick-contribution-tips-and-guidelines)
 * [Community Guidelines](#docker-community-guidelines)
 
+## Development environment
```

Comment:
Another option is cloning the CLI repo inside your global `GOPATH`. Although I guess we will move to go modules soonish?

```shell
mkdir -p $GOPATH/src/github.com/docker/
cd $GOPATH/src/github.com/docker
git clone git@github.com:docker/cli.git
```